OutOfSchool/OutOfSchool.WebApi/Util/MappingProfile.cs

```diff
@@ -519,6 +519,7 @@ public MappingProfile()
             .ForMember(dest => dest.WithDisabilityOptions, opt => opt.Ignore())
             .ForMember(dest => dest.AvailableSeats, opt => opt.Ignore())
             .ForMember(dest => dest.TakenSeats, opt => opt.Ignore())
+            .ForMember(dest => dest.CompetitiveSelection, opt => opt.Ignore())
```

Comment:
more of a question to BA: should we show that the workshop has competitive selection to users on 'applications' screen so he/she won't need to click every workshop to find out